systers_portal/templates/account/email.html

```diff
@@ -0,0 +1,58 @@
+{% extends "base.html" %}
+
+{% load account %}
+
+{% block title %} - Email{% endblock %}
+
+{% load crispy_forms_tags %}
+
+{% block content %}
+
+  <h1 class="login form-signin">E-mail Addresses</h1>
```

Comment:
There is no need to indent that block.  Please remove the `login` and `form-signin` CSS classes. They have nothing to do with this page. Allow the width to be 100%, text aligned to left.